Dockerfile.original

```
@@ -0,0 +1,103 @@
+#    - Uses cache mounts for Hex/Rebar and Mix builds to speed up rebuilds.
+###############################################################################
+FROM elixir:1.18.3-otp-27-slim AS deps
+
+WORKDIR /app
+
+# Set Mix environment and a default application version (overridable via build-arg)
+ENV MIX_ENV=prod \
+    APP_VERSION=0.1.0-docker
+
+# Install only the build tools we need
+RUN apt-get update \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)